### PR TITLE
extending mandatory fields for 0x4652000000000017

### PR DESCRIPTION
**Pull request by magdignomad**

### CODE REVIEW

File: doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ For French law fulfullment a  ManagerId is required, if the receipt is sent as t
 
 | **Field Name** | **Data Type** | **Default Value Mandatory Field** | **Description**                                                       | **Version** |
 |----------------|---------------|-----------------------------------|-----------------------------------------------------------------------|-------------|
-| `ManagerId`	                |`String`<br>Max 1k |	empty-string<br>optional<br>mandatory in training mode | Identification of the user, who started the training mode. | 1.2      |
+| `ManagerId`	  |`String`<br>Max 1k |	empty-string<br>optional<br>mandatory in training mode | Identification of the user, who started the training mode. | 1.2      |
+| `from`	        |`long`             | 0<br>mandatory for receiptcase `0x4652000000000017` | date/time in ticks for the beginning of the journal export.   | 1.2      |
```
Comment (TSchmiedlechner) [inline doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md:23]: Should be `FromTimestamp`

File: doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md [hunk 2]
```diff
@@ -19,7 +19,9 @@ For French law fulfullment a  ManagerId is required, if the receipt is sent as t
...
+| `to`	        |`long`             | 0<br>mandatory for receiptcase `0x4652000000000017` | date/time in ticks for the end of the journal export.<br>Timespan to `from` must be less than one year.   | 1.2      |
```
Comment (TSchmiedlechner) [inline doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md:24]: Shoule be `ToTimestamp`

File: doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md (hunk 1 above)
Comment (magdignomad) [reply doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md]: changed

File: doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md (hunk 2 above)
Comment (magdignomad) [reply doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md]: changed

**Review by TSchmiedlechner — APPROVED** [review]

**Pull request merged**